.github/workflows/python_unit_tests.yml

```diff
@@ -0,0 +1,25 @@
+name: Python Unit Tests
```

Comment:
Just to sanity-check, have you been able to test this somehow?  I'm not very familiar with GitHub workflows, so not sure how you go about vetting them before committing.

Reply:
yep - it actually runs as soon as you add a file - so you can see in the results below